Source/VisualStyle.swift

```diff
@@ -105,7 +110,7 @@ extension VisualStyle {
 public class DefaultVisualStyle: VisualStyle {
 
     private let alertStyle: AlertControllerStyle
-    init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
+    public init(alertStyle: AlertControllerStyle) { self.alertStyle = alertStyle }
```

Comment:
Yep. I had to add my own initializers. For the current moment I used this library with my customizations.

Anyway here I'm confused a bit, because we have a class several methods of which use private variable that initialized by internal init. I didn't try who this work, but I have assumption that swift will crash.